resource-manager/src/main/java/org/ihtsdo/otf/resourcemanager/ResourceManager.java

```diff
@@ -105,7 +105,7 @@ public boolean doesObjectExist(final File resource) throws IOException {
 				String resourcePath = resource.getPath().replaceAll("\\\\", "/");
 				resourcePath = (configPath != null ? configPath : "") + resourcePath;
 				String bucketName = resourceConfiguration.getCloud().getBucketName();
-				return ((S3ResourceLoader) resourceLoader).getS3Client().doesObjectExist(bucketName, resourcePath);
+				return ((S3ResourceLoader) resourceLoader).getAmazonS3().doesObjectExist(bucketName, resourcePath);
```

Comment:
It won't work with ResourceLoader created by spring cloud aws via auto wired. We should try our best not to use externally configured AWS credentials.